storagemodule.py

```diff
@@ -0,0 +1,57 @@
+#Question: Do you want me to make this whole thing a module or lots of submodules ? 
+
+#Import libaries
+import os
+import shutil
+import earthpy as et
+
+#Make a directory in any os system's home directory 
+et.io.HOME
+#os.path.exists(et.io.HOME) #a check to see if making it was successful 
+aronspath=os.path.join(et.io.HOME, "arons-kit")#Make the main directory called arons-kit in home directory 
+
+os.mkdir(aronspath)# need to do some conditional statements here if the directory already exists.
+os.chdir(aronspath)#change path to this directory 
+mainpath=os.getcwd()#fetches user's working directory 
+
+#Could make this a function so its a bit cleaner ? 
+# Create the sub-directory Staging
+stagingpath=os.path.join(mainpath,'Staging') # staging directory path 
```

Comment:
I'm confused what was the point of creating the aarons-kit folder if you're making a staging directory path?